main/src/main/scala/sbt/Defaults.scala

```diff
@@ -1098,12 +1105,10 @@ object Defaults extends BuildCommon {
   }
 
   /** Implements `cleanFiles` task. */
-  def cleanFilesTask: Initialize[Task[Vector[File]]] =
```

Comment:
Can't get rid of this, for binary compatibility (MiMa-enforced) reasons.

Reply:
Does f486d9d fix it?

Reply:
Travis CI will let us know.

Reply:
Travis is green ✅